app/Http/Middleware/DataClerkMiddleware.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class DataClerkMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if(auth()->user()->data_clerk()) {
```

Comment:
```suggestion
        if(auth()->user()->dataClerk()) {
```

Follow Laravel conventions. Methods and variables should be in camelCase